### ISSUE TITLE
Accessibility issue: ctrl pageup or pagedown screen reader did not read the title of the tab

### ISSUE THREAD

**Issue by ghost** [notepad-plus-plus/notepad-plus-plus#10724]
### Description of the Issue

Hello,
This is a screen reader user 
Since upgrading to v 8.1.x, when I switch to different tab using ctrl PageUp or PageDown, my screen reader no longer auto speak the title of the tab.

I am currently using Windows 10, npp 8.1.9 64bit
I am using screen reader NVDA.
Thanks.

### Steps to Reproduce the Issue

1. open npp with two or more tabs being opened
2. press either PageUp or PageDown to switch to another tab
3. screen reader NVDA did not say anything

However, if I switch tab using ctrl+tab, my screen reader can speak the switched tab
1. press ctrl+tab
2. my screen reader anounce a dialogue with a list of opened tabs
3. if i keep tab, screen reader move the focus on to next tiem on the list and will anounce the name of the tab
4. if i release all the keys, the tab is switched based on what is last selected on the list

### Expected Behavior

after switching the tab using ctrl+page up or page down, screen reader will immediately report the title of the new tab.

### Actual Behavior

after switching tab using ctrl + page up or page down, screen reader say nothing.

### Debug Information

Notepad++ v8.1.9   (64-bit)
Build time : Oct 21 2021 - 23:37:53
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Home (64-bit) 
OS Version : 2009
OS Build : 19043.1288
Current ANSI codepage : 950
Plugins : mimeTools.dll NppConverter.dll NppExport.dll

**Comment by donho** [comment]
@sine-w 
Is it still an issue for v8.6?

**Comment by sine-w** [comment]
Yes, I just updated my npp to 8.6 64bit, and the problem is still here.
Let me update the exact steps of the problem in the main post.
